docs/proposals/05-versioning-policy.md

```diff
@@ -7,26 +7,21 @@
 
 ## Motivation
 
-The Kubernetes community releases **minor** versions roughly every three months and usually maintains **three minor** versions (the actual and the last two) with bug fixes and security updates. Patch releases are done more frequently. Gardener creates plain vanilla Kubernetes clusters provided by the community and therefore relies on the community to provide bug fixes and security updates. This means in particular, that we do **not** back-port bug fixes or security updates to older Kubernetes versions which are not maintained by the community anymore.
-
-However we are aware that there is demand from our stakeholders to stay on a specific Kubernetes version for a longer period of time. For that reason, Gardener offers also older Kubernetes versions on a voluntary basis which are not maintained by the community anymore until we learn of issues (security vulnerabilities, bugs, operational effort, etc.) that make it "unbearable" to keep that version available. There are no hard rules here and this decision will be taken on a case by case basis by the Gardener team. The expiration process will follow clear rules, however (see below).
+The Kubernetes community releases **minor** versions roughly every three months and usually maintains **three minor** versions (the actual and the last two) with bug fixes and security updates. Patch releases are done more frequently. Operators of Gardener should be able to define their own Kubernetes version policy. This GEP suggests the possibility for operators to classify Kubernetes versions, while they are going through their "maintenance life-cycle".
 
 ## Kubernetes Version Classifications
 
-️️️️️️⚠️ **_NOTE:_**
-If you opt-out of automatic cluster updates, it is the responsibility of the cluster owner to update the clusters in case of known security vulnerabilities.
+An operator should be able to classify Kubernetes versions differently while they go through their "maintenance life-cycle", starting with **preview**, **supported**, **deprecated**, and finally **expired**. This information should be programmatically available in the `cloudprofiles` of the Garden cluster as well as in the Dashboard. Please also note, that Gardener keeps the control plane and the workers in lock-step, i.e. on the same Kubernetes version.
```

Comment:
What does "Gardener keeps the control plane and the workers in lock-step, i.e. on the same Kubernetes version." mean?

Reply:
API server, controller managers, etc. in the seed and the kubelet on the shoot workers. That's different from, let's say, GKE where the control plane can be of a different version than the nodes, I believe.

Reply:
Ah, okay, yes, sure, thank's for clarifying!